arch/s390/s390_functions.h

```diff
@@ -0,0 +1,16 @@
+#ifndef S390_FUNCTIONS_H_
```

Comment:
Missing the comment mentioning what the file does as seen on similar files. All new files should mention that the code is licensed under zlib license.